public/styles/component.scss

```diff
@@ -120,20 +120,25 @@ kbn-dis doc-table .kbnDocViewer__warning {
     flex-grow: 0;
 }
 
-.osButtonsStyle {
-    display: grid;
-    grid-template-columns: repeat(5, 1fr);
-    grid-gap: 10px;
+.message {
+    margin-top: 10px;
+    display: flex;
+    flex-direction: row;
 }
 
-.osButtonsStyleMac {
+.euiButtonGroup__buttons {
     display: grid;
-    grid-template-columns: repeat(3, 1fr);
+    grid-template-columns: repeat(5, 1fr);
     grid-gap: 10px;
+    padding: 0 3px;
+    box-shadow: none;
 }
 
-.message {
-    margin-top: 10px;
+.flex {
     display: flex;
-    flex-direction: row;
+}
+
+.euiButtonGroup--medium .euiButtonGroupButton:not(:first-child), .euiButtonGroup--small .euiButtonGroupButton:not(:first-child) {
+    margin-left: 0 !important;
+
 }
```

Comment:
**question**: is this a class of the @elastic/eui? If it does, we should ensure only to affect the UI of the Wazuh plugin. This could be modifying the styles of plugins of the platform.

Reply:
**thought**: For another hand, this pull request has as its base the branch `4.4-2.3-wzd`, which is a plugin compatible with Wazuh dashboard based on OpenSearch Dashboards 2.3.0. This version of OpenSearch is using a forked UI library, https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.3.0/package.json#L111, and it seems the class names are prefixed by `oui` instead of `eui` https://github.com/opensearch-project/oui/blob/1.0.0/src/components/button/button_group/_button_group_button.scss#L122. Is these styles applying?

Reply:
> **thought**: For another hand, this pull request has as its base the branch `4.4-2.3-wzd`, which is a plugin compatible with Wazuh dashboard based on OpenSearch Dashboards 2.3.0. This version of OpenSearch is using a forked UI library, https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.3.0/package.json#L111, and it seems the class names are prefixed by `oui` instead of `eui` https://github.com/opensearch-project/oui/blob/1.0.0/src/components/button/button_group/_button_group_button.scss#L122. Is these styles applying?

I was testing the branch and it is working. It seems that the UI library contains components from @elastic/eui and oui components.

Anyways, we should ensure that the modified styles only happen in the Wazuh plugin.

Reply:
Fixed in this [PR](https://github.com/wazuh/wazuh-kibana-app/pull/4837)